client/src/components/FileStatusIcon/FileStatusIcon.js

```diff
@@ -0,0 +1,112 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import i18n from 'i18n';
+import { UncontrolledTooltip } from 'reactstrap';
+
+/**
+ * File status icon (e.g. restricted access) with a tooltip
+ */
+class FileStatusIcon extends Component {
+  constructor(props) {
+    super(props);
+    this.cssClass = 'file-status-icon';
```

Comment:
Is this meant to be a prop? It might as well be, otherwise you can just hardcode it, or make it `const cssClass` declared above the class.

Reply:
Updated to use `const cssClass`